tests/Core/Components/Progress/FluentSpinnerTests.razor

```diff
@@ -0,0 +1,72 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using Xunit;
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Samples;
+@inherits TestContext
+@code
+{
+    public FluentSpinnerTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddFluentUIComponents();
+    }
+
+    [Fact]
+    public void FluentSpinner_Default()
+    {
+        // Act
+        var cut = Render(@<FluentSpinner />);
+
+        // Assert
+        cut.MarkupMatches("<fluent-spinner />");
+    }
+
+
+    [Fact]
+    public void FluentSpinner_AppearanceInverted()
+    {
+        // Act
+        var cut = Render(@<FluentSpinner AppearanceInverted="true" />);
+
+        // Assert
+        cut.MarkupMatches("<fluent-spinner appearance='inverted' />");
+    }
+
+    [Theory]
+    [InlineData(SpinnerSize.Tiny)]
+    [InlineData(SpinnerSize.ExtraSmall)]
+    [InlineData(SpinnerSize.Small)]
+    [InlineData(SpinnerSize.Medium)]
+    [InlineData(SpinnerSize.Large)]
+    [InlineData(SpinnerSize.ExtraLarge)]
+    [InlineData(SpinnerSize.Huge)]
+    [InlineData((SpinnerSize)999)]
+
```

Comment:
Should there not be a InlineData `null` test as well

Reply:
Yes :-)
Done